packages/eslint-config/lib/index.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
+
+const config = {
```

Comment:
Actually there's a lot in here we probably don't need, like the `globals`, `extends`, and `overrides` properties. All of these will be inherited from the eslint-config-next package. We can probably get away with only actually setting `env` and `rules` 🙂